block-node/blocknode-filesystem-s3/build.gradle.kts

```diff
@@ -14,4 +14,7 @@
  * limitations under the License.
  */
 
-plugins { id("com.hedera.hashgraph.blocknode.conventions") }
+plugins {
+    id("com.hedera.hashgraph.blocknode.java")
+    id("com.hedera.hashgraph.blocknode.publish")
```

Comment:
I started to unify a bit more how we name things in the Gradle plugins. Right now, some things have a `convention` suffix and some don't. I would drop the suffix, as everything here is a "convention" in the Gradle sense.

I now have for each set of projects a ".java" (saying this is a Java project with specific conventions) and a ".publish" (this is a published library with specific conventions)